uhabits-core/src/jvmTest/java/org/isoron/uhabits/core/utils/DateUtilsTest.kt

```diff
@@ -118,17 +118,24 @@ class DateUtilsTest : BaseUnitTest() {
         assertThat(arrayOf("Sat", "Sun", "Mon", "Tue", "Wed", "Thu", "Fri"), equalTo(longWeekdayNames))
     }
 
+    private fun getCalendarUTC(year: Int, month: Int, dayOfMonth: Int): GregorianCalendar {
```

Comment:
If we use this only in a single, we might as well put it as a local function inside that test, IMHO.

Reply:
sure

on Sunday I will do it

Reply:
Force pushed, and tested on APIs 28, 29, 30, everything passes locally.